src/DotNetWorker.OpenTelemetry/DotNetWorker.OpenTelemetry.csproj

```diff
@@ -6,7 +6,7 @@
         <AssemblyName>Microsoft.Azure.Functions.Worker.OpenTelemetry</AssemblyName>
         <RootNamespace>Microsoft.Azure.Functions.Worker.OpenTelemetry</RootNamespace>
         <MajorProductVersion>1</MajorProductVersion>
-        <MinorProductVersion>0</MinorProductVersion>
+        <MinorProductVersion>1</MinorProductVersion>
         <PatchProductVersion>0</PatchProductVersion>
         <VersionSuffix>-preview1</VersionSuffix>
```

Comment:
Do we need to adjust the VersionSuffix here? This feels like it could be a 1.0.0-preview2